templates/Dockerfile-deb.tmpl

```diff
@@ -89,8 +90,12 @@ RUN curl -s https://packagecloud.io/install/repositories/citusdata/community/scr
 COPY make_pg_buildext_parallel.patch /
 RUN patch `which pg_buildext` < /make_pg_buildext_parallel.patch
 
+# Added for pg15 beta package support. Without this pg_buildext gives below error
```

Comment:
I do not see any error message here. I guess you can remove part of this comment as it is quite self explanatory already.

```suggestion
# Added for pg15 beta package support.
```